pallets/spin/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
```

Comment:
Please mention somewhere the upstream version (commit) from which we are copying the code.